pegasus_wrapper/key_value.py

```diff
@@ -60,20 +61,44 @@ class KeyValueTransform(Protocol):
     """
 
     def __call__(
-        self,
-        input_zip: KeyValueStore,
-        *,
-        workflow_builder: WorkflowBuilder,
-        output_locator: Optional[Locator] = None,
+        self, input_zip: KeyValueStore, *, output_locator: Optional[Locator] = None
     ) -> KeyValueStore:
         """
         A function which adds jobs to *workflow_builder* to transform *input_zip*
         into another `KeyValueZip`.
         """
 
 
+@attrs(frozen=True)
```

Comment:
has stuff from https://github.com/isi-vista/vista-pegasus-wrapper/pull/32

Reply:
Yes, this builds off that one and shouldn't be merged until that one is merged.